markdown/org/docs/patterns/lunetius/cutting/en.md

```diff
@@ -4,6 +4,11 @@ title: Cutting
 
 The way to cut Lunetius is going to depend on what fabric you have chosen to use. If you have enough fabric width you can cut it _on the fold_. If you don’t, you can cut two halves and later sew them together. To not have to later finish the centre back edge you can cut with the centre back edge on the selvage.
 
+### Materials
```

Comment:
Same here. No need for all them subtitles. When in doubt, check Aaron: https://freesewing.org/docs/patterns/aaron/cutting/